### ISSUE TITLE
Error: Scene validation errors

### ISSUE THREAD

**Issue by bryshen** [thanacus/alchemist-import#3]
I updated to Foundry V12 and I'm getting this error when importing any map:

Scene validation errors:
  walls: 
    null: 
      move: 1 is not a valid choice
      sound: 1 is not a valid choice
    null: 
      move: 1 is not a valid choice
      sound: 1 is not a valid choice
-----------------------------------
This error repeats over and over and the map is not created.

However, the image file is added to the target directory correctly.

**Comment by thanacus** [comment]
@bryshen can you post the json and jpeg you're using? I'm able to import maps just fine over here on V12

**Comment by Nmarinsiruela** [comment]
[Aldea de los Picos Caidos 3.json](https://github.com/user-attachments/files/16208773/Aldea.de.los.Picos.Caidos.3.json)

![Aldea de los Picos Caidos 4](https://github.com/user-attachments/assets/ae3f0364-cc89-4d2f-bc86-0e234d25b122)

I'm currently suffering the same issue here! Funny thing, it did work for me like 15 days ago, and I tried to upgrade to the latest version to see if that help. Spoilers: it did not!

BTW: Love the module! Been using it a ton, and this is the first time I've actually had any kind of issue!

**Comment by thanacus** [comment]
Thanks for the file! The good news is: I can reproduce your issue by making a map with builtin walls. The bad news is: the data that Dungeon Alchemist is exporting is not fully compatible with V12, it seems.

Dungeon Alchemist export values look like this for a "wall" object:
```
    {
      "c": [
        1727,
        2099,
        1727,
        1952
      ],
      "move": 1,
      "sense": 1,
      "sound": 1,
      "door": 0
    },
```

Whereas when you export a map with walls from Foundry V12 it looks like this: 
```
"walls": [
    {
      "light": 20,
      "sight": 20,
      "sound": 20,
      "move": 20,
      "c": [
        538,
        463,
        538,
        325
      ],
      "_id": "NYr9nBmwMs4Ob4va",
      "dir": 0,
      "door": 0,
      "ds": 0,
      "threshold": {
        "light": null,
        "sight": null,
        "sound": null,
        "attenuation": false
      },
      "flags": {}
    },
```

I changed all your "move": 1 etc. to "move": 20 and it suddenly imports normally. This could be a fix in general for importing from DA as even going through the normal process will likely run into this issue. I could possibly implement a patch, but I have no idea what the valid values should be (1 vs 20 means what?) so it'll take some time.

**Comment by thanacus** [comment]
Do me a favor and check out 1.0.8 update! I compared your raw JSON vs a direct import/export of your JSON and came up with a mapping for the values that imports cleanly now.

**Comment by thanacus** [comment]
> Do me a favor and check out 1.0.8 update! I compared your raw JSON vs a direct import/export of your JSON and came up with a mapping for the values that imports cleanly now.

@bryshen @Nmarinsiruela

**Comment by Nmarinsiruela** [comment]
Will do! And yes, it now works, great! Thank you for that quick fix! I'm not sure of what the heck those numbers should do, but it does seem this work!

**Comment by bryshen** [comment]
It's working for me with the new update. Thanks so much!

**Comment by thanacus** [comment]
Fixed with v1.0.8
